public/javascript/ep-card-filter.js

```diff
@@ -194,9 +199,45 @@ $(document).ready(function(){
       window.cards.saveState();
     });
 
-    $('[data-section-toggle]').on('click', function(){
-      var section = $(this).attr('data-section-toggle');
+    $('.js-show-facet').on('change', function(){
```

Comment:
Is it not preferable to maintain consistency with all the other event bindings elsewhere, which all use `.on()`?